### DISCUSSION TITLE
Can Python UI widget PerspectiveWidget be set color like javascript?

### DISCUSSION THREAD

**Discussion by sunphoenix** [perspective-dev/perspective#1975] [Q&A] [answered]
## Support Question

I plan to make a python dashboard based on PerspectiveWidget. Does PerspectiveWidget has the full functionality like javascript library?

I found the folloing repo is quite cool:
https://github.com/sc1f/perspective-order-book/

Can I make it with python PerspectiveWidget library?

**Comment by timkpaine** [comment, marked as answer]
You can make a very similar dashboard as the one from that repo using perspective widgets. Some things that you cannot currently do with perspective widget:

- there is no support for perspective workspace in jupyter yet
- there is no ability to register custom plugins for perspective viewer (not done in that example, but in several of the other ones for custom rendering in charts and tables)
